controllers/admin/AdminCartsController.php

```diff
@@ -480,7 +480,7 @@ public function ajaxProcessUpdateQty()
                     $errors[] = $this->trans('You already have the maximum quantity available for this product.', array(), 'Admin.Catalog.Notification');
                 } elseif ($qty_upd < 0) {
                     $minimal_qty = $id_product_attribute ? Attribute::getAttributeMinimalQty((int)$id_product_attribute) : $product->minimal_quantity;
-                    $errors[] = sprintf(Tools::displayError('You must add a minimum quantity of %d', false), $minimal_qty);
+                    $errors[] = sprintf($this->trans('You must add a minimum quantity of %d', array(), 'Admin.Catalog.Notification'), $minimal_qty);
```

Comment:
There is a lot of code that keeps a sprintf around a $this->trans in many places. If it is not allowed, please give me a link to the correct rules. Also PrestaShop team should correct many other places and stop using the code incorrectly. People are looking on PS code.